modules/installation-configuration-parameters.adoc

```diff
@@ -1032,6 +1032,53 @@ If you deploy to a custom subnet, you cannot specify an external DNS server to t
 
 |A UUID as a string. For example, `fa806b2f-ac49-4bce-b9db-124bc64209bf`.
 |====
+
+[id="installation-configuration-parameters-failure-domains-osp_{context}"]
+== {rh-openstack} parameters for failure domains
+
+:FeatureName: {rh-openstack} failure domains
+[IMPORTANT]
+====
+[subs="attributes+"]
+{FeatureName} is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+// Undefine {FeatureName} attribute, so that any mistakes are easily spotted
+:!FeatureName:
+
+You can distribute the control plane across availability zones by using failure domains. 
+
+To define failure domains, add items to the `platform.openstack.failuredomains` list that contain the following parameters:
```

Comment:
I am not familiar with the documentation format. Is this paragraph in the context of machine-pools?

The absolute path to `failureDomains` is `controlPlane.platform.openstack.failureDomains`. The failure domains, in the current (TechPreview) implementation, are defined per-machinepool and do not apply cluster-wide (note that this may change before GA, which is targeted at OCP v4.14).

Reply:
It definitely is now. 🙃

Reply:
```suggestion
To define failure domains for a machine pool, add items to the `controlPlane.platform.openstack.failureDomains` list that contains the following parameters:
```